src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -367,6 +440,47 @@ export class RuleEditor extends LinkedComponent {
                 </Section.Container>
               ))
             }
+            <Section.Container collapsable={false}>
+              <Section.Header>{t('rules.flyouts.ruleEditor.actions.actions')}</Section.Header>
+              <ToggleBtn
+                value={formData.actionEnabled}
+                onChange={this.onActionToggle} >
+                {formData.actionEnabled ? t('rules.flyouts.ruleEditor.actions.on') : t('rules.flyouts.ruleEditor.actions.off')}
+              </ToggleBtn>
+              {
+                formData.actionEnabled && <p>{t('rules.flyouts.ruleEditor.actions.sendEmailOnTrigger')}</p>
+              }
+              {
+                actionLinks.map((action, idx) => (
+                  <div key={formData.actions[idx].key}>
+                    <Section.Content>
+                      {
+                        formData.actionEnabled &&
```

Comment:
It doesn't seem like you need this check inside the `map`. Better to put this check before the `actionLinks.map`
```
{
  formData.actionEnabled &&
  actionLinks.map((action, idx) => (
}
```

Reply:
Moved.